File: demo/src/main/java/cn/bingoogolapple/progressbar/demo/DefaultActivity.java
package cn.bingoogolapple.progressbar.demo;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class DefaultActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_default);
        setTitle("系统自带 ProgressBar 支持的样式");
    }
}

File: demo/src/main/java/cn/bingoogolapple/progressbar/demo/DefaultActivity.kt
package cn.bingoogolapple.progressbar.demo

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class DefaultActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_default)
        title = "系统自带 ProgressBar 支持的样式"
    }
}
